mesh/accountsettings/accountsettings.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS AccountSettings(
+	settingID INT PRIMARY KEY AUTO_INCREMENT,
+    accountID INT,
+    isVerified BOOL NOT NULL DEFAULT FALSE,
```

Comment:
PR #143 uses `BOOLEAN` and `DEFAULT 0`, while you use `BOOL` and `DEFAULT FALSE`. They are semantically-identical, but I think we should be consistent for clarity. `FALSE` is more readable than `0` imo, but `BOOLEAN` vs. `BOOL` probably doesn't matter.